include/samplog/LogLevel.h

```diff
@@ -14,8 +14,15 @@ enum samplog_LogLevel
 	INFO = 2,
 	WARNING = 4,
 	ERROR = 8,
+	FATAL = 16,
```

Comment:
FATALs are useful at server startup at the very least - if some required files are missing or the version can't be determined for example.  I'm not really sure that doing anything more than recording this fact is in scope for a logging system.  It just logs, and while fatal errors shouldn't happen, if they do they should be logged accordingly.

The plugin already handles flushing logs in a range of situations and server shut downs, so even if it did close from a fatal error, this system would still clean up correctly from existing code.